### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by cesy** [Growstuff/growstuff#3308]
Heroku have asked for our code of conduct to meet their latest criteria for charity hosting. We already have https://www.growstuff.org/policy/community which covers some overlapping ground but is mostly focused on website users - it would be good to have something for the OSS project, to cover issues, pull requests, etc.

https://opensource.guide/code-of-conduct/ has some good guidance. https://www.contributor-covenant.org/ is usable by other projects, and v2.1 seems reasonable. For the contact details, we can use our usual info@ email address that goes to all maintainers and is already public in other places and gets spam already, and individual maintainers can be approached if there's an issue involving another maintainer - most of us have at least one public method of private contact, whether email or social media DM.

@CloCkWeRX @Br3nda @maco @pozorvlak are you happy with that, or shall we discuss more on private email or here? Once we're agreed, we can add it to our README and as a CODE_OF_CONDUCT file on Github.

**Comment by maco** [comment]
I’m not at my computer so can’t compare at the moment, but GNOME has a
great one that Sage Sharp consulted on, adapted from the Ada Initiative and
other ones.

**Comment by pozorvlak** [comment]
I'm fine with that. The Contributor Covenant document isn't exactly what
I'd have written, but I think the advantages of standardisation outweigh
the benefits of tweaking it.

**Comment by CloCkWeRX** [comment]
I whacked the merge button. I'm fine with the broad idea and language. 

I'm a bit less comfortable that we're doing this as a somewhat performative move for what boils down to funding purposes because an organisation wants to virtue signal; while itself not necessarily doing... much.

For example; public statements ... three?

https://blog.heroku.com/code_of_conduct - 2013

Then:
https://blog.heroku.com/tags/pride
https://blog.heroku.com/tags/values

Tangible actions that help marginalised groups in a specific measurable way... unclear? Cutting off open source projects from their resources for commercial reasons is the biggest thing I know of, and I don't quite see how that helps those groups participate.


I am not familar with how they interact with various communities beyond a brief google and this project/similar projects.
Perhaps there are things I am not across that they do that are important and useful I'm not aware of.


That said, I would much prefer if we evolved our own policy here, based on the history of who has contributed, their values and acts.

A really good tangible example of this: the nudge to add yourself to the contributors list when you add code via the automations. https://github.com/Growstuff/growstuff/blob/dev/CONTRIBUTORS.md#contributors
It's a practical acknowledgement that you decided to add to a community. It's not earth shattering, but it is *recognition*, and very importantly it applies no matter your shape, size, ability to do handstands, etc; and it's inclusive.

That feels more like us, than Heroku demanding we publish values which are basically self evident if you participate in our community.
